streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamImplTest.java

```diff
@@ -2455,14 +2482,13 @@ public void shouldSupportKeyChangeKTableFromKStream() {
         final String output = "output";
 
         builder.stream(input, consumed)
-            .map((key, value) -> new KeyValue<>(key + "-", value))
-            .toTable()
+            .map((key, value) -> new KeyValue<>(key.charAt(0) - 'A', value))
```

Comment:
Changing this test, to also change the key _type_ to make sure we can pass in serdes correctly.